int32[] pos
float32[] accel
int16[] encoder
